### PR TITLE
Updated charts of the human-task explorer and bpmn-explorer

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: components/humantask/humantask-explorer-web/src/web/template/statsView.jag [hunk 1]
```diff
@@ -10,34 +10,55 @@
...
+          google.load("visualization", "1", {packages:["corechart"]});
+          google.setOnLoadCallback(drawChart);
 
-        var chart = new google.visualization.BarChart(document.getElementById('taskStatusCountBar'));
-        chart.draw(data, options);
-      }
-    </script>
+          var dataArr = [['Status of the task', 'Task Count']];
+           <% for(var j = 0; j < statusArrayBar.length; j++){ %>
+
```
Comment (sinthuja) [inline components/humantask/humantask-explorer-web/src/web/template/statsView.jag]: Remove unnecessary spaces.

File: components/humantask/humantask-explorer-web/src/web/template/statsView.jag [hunk 2]
```diff
@@ -10,34 +10,55 @@
...
+                dataArr.push(['<%=statusArrayBar[j][0]%>',<%=statusArrayBar[j][1]%>]);
```
Comment (sinthuja) [inline components/humantask/humantask-explorer-web/src/web/template/statsView.jag]: Formatting issue, there should be a space after and before comma.

File: components/humantask/humantask-explorer-web/src/web/template/statsView.jag (hunk 1 above)
Comment (NatashaWso2) [inline components/humantask/humantask-explorer-web/src/web/template/statsView.jag]: Fixed with commit ad9af2953e9206cfced0e8956b72c4fe548f8cd3

File: components/humantask/humantask-explorer-web/src/web/template/statsView.jag (hunk 2 above)
Comment (NatashaWso2) [inline components/humantask/humantask-explorer-web/src/web/template/statsView.jag]: Fixed with commit ad9af2953e9206cfced0e8956b72c4fe548f8cd3

File: components/bpmn/bpmn-explorer-web/src/web/template/statsView.jag [hunk 3]
```diff
@@ -22,53 +22,79 @@
...
+              if (dataArr[i][1] > max) {
+                max = dataArr[i][1];
+              }
+              
+             if(dataArr[i][1]>0){
+                count++;
+              }     
+            }
+           
+
```
Comment (jranabahu) [inline components/bpmn/bpmn-explorer-web/src/web/template/statsView.jag]: Remove extra new lines

Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/template/statsView.jag]: Fixed with commit 130406a

**Pull request merged**